src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CcdSearchService.java

```diff
@@ -109,8 +109,13 @@ public SearchResult searchForCasesWithQuery(final int from,
             .sort("data.issueDate", ASC)
```

Comment:
whilst we're changing this can you remove this

Reply:
Will do 👍